packages/jest-types/src/Config.ts

```diff
@@ -184,7 +184,7 @@ export type InitialOptions = Partial<{
   preprocessorIgnorePatterns: Array<Glob>;
   preset: string | null | undefined;
   prettierPath: string | null | undefined;
-  projects: Array<Glob>;
+  projects: Array<Glob> | Array<Omit<InitialOptions, keyof GlobalConfig>>;
```

Comment:
self reference works? if not we can have one type without projects, and then one that extends it